static/css/progress-bar.css

```diff
@@ -0,0 +1,120 @@
+.progress-bar {
+    height: 3px;
+    margin: 0px;
+    -webkit-appearance: none;
+}
+  
+/*progress support*/
+.slider-progress {
+    --range: calc(var(--max) - var(--min));
+    --ratio: calc((var(--value) - var(--min)) / var(--range));
+    --sx: calc(0.5 * 13.53px + var(--ratio) * (100% - 13.53px));
+}
+  
+.progress-bar:focus {
+    outline: none;
+}
+  
+/*webkit*/
+.progress-bar::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    width: 13.53px;
```

Comment:
noooo, pls noooo, Don't use float numbers for px, browser engine can count it wrong